components/UI/wallets.tsx

```diff
@@ -57,13 +57,13 @@ const Wallets: FunctionComponent<WalletsProps> = ({
         {connectors.map((connector) => {
           if (connector.available()) {
             return (
-              <div className="mt-5 flex justify-center" key={connector.id()}>
+              <div className="mt-5 flex justify-center" key={connector.id}>
                 <Button onClick={() => connectWallet(connector)}>
                   <div className="flex justify-center items-center">
-                    <WalletIcons id={connector.id()} />
-                    {connector.name() === "Argent Web Wallet"
-                      ? "Login with Email"
-                      : `Connect ${connector.name()}`}
+                    <WalletIcons id={connector.id} />
+                    {connector.id === "braavos" || connector.id === "argentX"
+                      ? `Connect ${connector.name}`
+                      : "Login with Email"}
```

Comment:
The old way to do manage that is better imo because we let the possibility to have another wallet than argent and Braavos.

When here you say it's either argent and Braavos or web wallet.